src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -330,6 +330,24 @@ public function findChildrenAndContacts(int $id): ?AccountInterface
         }
     }
 
+    /**
+     * @return AccountInterface[]
+     */
+    public function findChildAccounts(int $id): array
+    {
+        try {
+            $qb = $this->createQueryBuilder('account')
+                ->where('account.parent = :accountId');
+
+            $query = $qb->getQuery();
+            $query->setParameter('accountId', $id);
+
+            return $query->getResult();
+        } catch (NoResultException $ex) {
+            return null;
```

Comment:
i think i would expect an empty array in this case

Reply:
You are right, that's better :slightly_smiling_face: Done.